projects/gnunet/default.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  pkgs,
+  sources,
+}@args:
+
+{
+  metadata = {
+    summary = "GNUnet is GNU's framework for secure peer-to-peer networking";
+    subgrants = [
+      "GNUnet"
```

Needs to exactly match the URL from the NLnet page.

```suggestion
      "gnunet"
```